### DISCUSSION TITLE
Questions about Pricing and Logging - Added features under Free Plan

### DISCUSSION THREAD

**Discussion by RaXorX** [rustdesk/rustdesk#6089] [Q&A]
@rustdesk I recently came across Rustdesk and had a few queries regarding the prices and logging/security

a) Seems like there are central servers from which Rustdesk relays things to other servers if necessary. So do the servers log any data/IPs - whether you have monitoring permissions to the relay servers to monitor any tampering or if it's running an altered repo or anything at all?

b) Hosting plans seem to be pretty much tailored towards corporate/company users. Will there ever be plans for home users? Most of the features from the hosting plans seem to be a bit irrelevant to home users.

I might be wrong but it seems as if that the data storage is being co-hosted on a separate service like nextcloud/vultr etc etc and since they only have packages that are tailored for corporates more. So you just streamline the process of hosting and maintaining it from your end.

If the servers (and relay servers) aren't logging anything and storing any data on the users, I can see why you couldn't provide Managed Access and contact book for free users as well which more or less requires an account to be created.

And if the storage isn't something co-hosted by other services, considering the storage usage will always be a lot, lot considerably less than the bandwidth costs for home users, could there be a possibility for cheaper plans for Home Users?

I as a home user mostly care about having Managed Devices and Contact Book. Both of which might require having an account. Most home users barely have more than a few 5-10 devices (single or family user). And even if there are a couple relatives/friends, hardly there are enough to fill up 20-30 entries even in the contact book. And even if there are more than that, I doubt this much of data per user would hardly ever cost more than the bandwidth costs that would come from the usage of rustdesk in itself. 

This was a gripe with me with Anydesk but seeing Rustdesk as an OSS and a community driven project, I would hope/think that Rustdesk could make a difference where others could not. 

Considering all that I'd love to see if Managed Access and Contact book features could be provided to home/free users as well, and if not, then maybe a cheaper plan could be introduced. Relatively cheaper considering that we don't necessarily need all the unnecessary things.

```
Documentation - Irrelevant 
2 licensed users - Irrelevant (I sense this is the number of accounts you can create under a single license? not really sure what that even means)
100 managed devices - (I am guessing this is the amount of devices you can link to a single account)
Web console - Irrelevant
Address book
Audit log - Irrelevant
Change ID - Irrelevant (most often reinstalling the software should change the ID )
Access control - (not sure what access control is for unless this is what Unattended access is dubbed as here)
Centralized settings - Irrelevant
Distributed relay servers - Irrelevant
```
Most of the things in that list is totally irrelevant to home users. And if that is not a possibility, or there are no plans of such, for one at least you could provide local contact book and managed devices so one could simply back that data up and out and restore in another session. And perhaps a feature that could sync these two things via the clients we tell it to.

Basically, it doesn't has to even actively sync since I see that centralized settings is something you're providing as paid. But we could sync these things to another client of our choice. That would be the best honestly. We could always just sync it to another device manually, and problem solved. 

So basically,
- if it could have a feature where it could sync the Settings (optional), Managed Devices (not sure how this will relate to since I imagine managed devices is something you need an account for - based on that this might be not possible), and Contact Book to another client,
- and have an option for backups (optionally periodic backups) would be pretty optimal

Provided if you can't provide these features for free users due to storage limitations or access.

And from there you could even just provide internal ways to sync these settings to a cloud service of choice, google drive, one drive, dropbox, nas, ftp etc etc

**Comment by Rusty-Weasel** [comment]
> So do the servers log any data/IPs -

Free Open Source RustDesk Server Database as example, and as you see, yes Ip-Adresses **can** be logged (default is on).
Pro version unkown in case of **closed source.**

![image](https://github.com/rustdesk/rustdesk/assets/61587283/e9c78708-bb8b-41cb-974a-49aba5f31bae)

further, without logging? how should he know?
![image](https://github.com/rustdesk/rustdesk/assets/61587283/56ff83a7-0fea-44f7-bda3-3d0a75ebfb68)

[Link to his post on linkedin](https://www.linkedin.com/posts/josephjacks_excited-to-share-that-rustdesk-has-now-crosssed-activity-7117662300727058432-Tzwh?utm_source=share&utm_medium=member_desktop)

> I would hope/think that Rustdesk could make a difference where others could not 

I thought that also for a time...invested a huge amount of time in it, but after pro version release my mind sadly changed.

**Comment by RaXorX** [comment]
> further, without logging? how should he know?


I'm aware IP logging might be necessary to keep track of user base, number of connections and usage statistics which is why I had asked about it. And even with all those logs per user would be more than the storage needed to save the contacts (contact book) and list of managed devices per user account.
